packages/grafana-ui/src/types/time.ts

```diff
@@ -11,11 +11,29 @@ export interface TimeRange {
   raw: RawTimeRange;
 }
 
+export interface AbsoluteTimeRange {
+  from: number;
+  to: number;
+}
+
 export interface IntervalValues {
   interval: string; // 10s,5m
   intervalMs: number;
 }
 
+export interface TimeZone {
+  raw: string;
+  isUtc: boolean;
+}
+
+export const parseTimeZone = (raw: string): TimeZone => {
+  return {
+    raw: raw,
```

Comment:
Minor: the extra `raw` can be removed
```suggestion
    raw,
```

Reply:
Fixed. Will push soon